src/components/accessories/admission/PatientAdmission.tsx

```diff
@@ -104,7 +114,7 @@ const PatientAdmission: FC = () => {
       />
       {createStatus === "FAIL" && (
         <div ref={infoBoxRef} className="info-box-container">
-          <InfoBox type="error" message={t("common.somethingwrong")} />
+          <InfoBox type="error" message={createError} />
```

Comment:
The identifier `createError` is not a function, so maybe we could avoid using a verb for naming that. I think `errorMessage` or something like that would be nice.